tests/e2e/utils/flows.js

```diff
@@ -73,6 +74,57 @@ const getCartItemExpression = ( productTitle, args ) =>
 const getRemoveExpression = () =>
 	'td[@class="product-remove"]//a[@class="remove"]';
 
+const PaymentsCustomerFlow = {
```

Comment:
Non blocker, but let's add a comment explaining why are we decided to keep Payments flows separately.

Reply:
Sure! Added in https://github.com/Automattic/woocommerce-payments/pull/816/commits/883b9ea063b58d3a71c4c9ee4241863c591252ff.